src/utils/Draft.js

```diff
@@ -134,22 +134,89 @@ function arrangePicks(picks) {
 const considerInCombination = (combination) => (card) =>
   card && arrayIsSubset(card.colors ?? card.details.color_identity ?? card.details.colors ?? [], combination);
 
+const findBestValueArray = (weights, pickNumPerc) => {
+  const x = weights.length * pickNumPerc;
+  const ceilX = Math.ceil(x);
+  const floorX = Math.floor(x);
+  if (x === floorX || ceilX === weights.length) {
+    return weights[floorX];
+  }
+  const xMod = x - floorX;
+  return xMod * weights[ceilX] + (1 - xMod) * weights[floorX];
+};
+
+const findBestValue2d = (weights, packNum, pickNum, initialState) => {
+  const packNumPerc = (packNum - 1) / initialState[0].length;
```

Comment:
can you rename all these variables to be more self descriptive. `packNumNormalized` etc

`xMod` is particularly bad to me